pages/dashboard/views/index.jsx

```diff
@@ -30,38 +22,39 @@ const Invitation = ({ token, establishment }) => (
 const Index = ({
   tabs,
   progress,
-  profile: {
-    firstName,
-    establishments,
-    invitations
-  }
+  profile
 }) => {
+
   return <Fragment>
     <Header
-      title={<Snippet name={firstName}>pages.dashboard.greeting</Snippet>}
+      title={<Snippet name={profile.firstName}>pages.dashboard.greeting</Snippet>}
     />
     <div className="govuk-grid-row">
       <div className="govuk-grid-column-full">
```

Comment:
This is wild, I appreciate that you may have not written this, but would you remind removing it? We have a single row, with a single full width column - this is the same as having _nothing at all_

Reply:
done